### PR TITLE
Fixed issue #13256 - Removing or modifying URL will now hide EVC

### PR DESCRIPTION
**Pull request by jasonrsadler**
## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).
- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review). (Ask a Brave employee to help if you cannot access this document.)

## Test Plan:
Fixes:  #13256

## Reviewer Checklist:

- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review) if one was not already requested.

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

@bsclifton @srirambv

### CODE REVIEW

**Comment by alexwykoff** [comment]
If not reviewed by eob, will be punted to later release.

File: app/renderer/components/navigation/urlBar.js [hunk 1]
```diff
@@ -64,7 +65,7 @@ class UrlBar extends React.Component {
   }
 
   maybeUrlBarTextChanged (value) {
-    if (value !== this.props.urlbarLocation) {
+    if (value !== this.props.urlbarLocation) {
```
Comment (diracdeltas) [inline app/renderer/components/navigation/urlBar.js]: please remove trailing whitespace. i think `npm run lint` should catch this

File: app/renderer/components/navigation/urlBar.js [hunk 2]
```diff
@@ -52,6 +52,7 @@ class UrlBar extends React.Component {
     this.lastVal = ''
     this.lastSuffix = ''
     this.isOnComposition = false
+    this.isChanging = false;
```
Comment (diracdeltas) [inline app/renderer/components/navigation/urlBar.js]: no trailing semicolon

**Comment by jasonrsadler** [comment]
Running lint did indeed catch it. Sorry about those old habits.

File: app/renderer/components/navigation/urlBar.js [hunk 3]
```diff
@@ -501,7 +513,7 @@ class UrlBar extends React.Component {
       <UrlBarIcon
         titleMode={this.props.titleMode}
       />
-      {this.showEvCert}
+      {this.showEvCert }
```
Comment (diracdeltas) [inline app/renderer/components/navigation/urlBar.js]: extra space before the bracket

Comment (jasonrsadler) [reply app/renderer/components/navigation/urlBar.js]: Linter didn't catch that one

File: app/renderer/components/navigation/urlBar.js [hunk 4]
```diff
@@ -52,6 +52,7 @@ class UrlBar extends React.Component {
     this.lastVal = ''
     this.lastSuffix = ''
     this.isOnComposition = false
+    this.isChanging = false
```
Comment (diracdeltas) [inline app/renderer/components/navigation/urlBar.js]: why not use `this.props.isActive` for this purpose?

**Comment by jasonrsadler** [comment]
Yes. That works. Didn't know about props.isActive.

File: app/renderer/components/navigation/urlBar.js [hunk 5]
```diff
@@ -489,7 +493,7 @@ class UrlBar extends React.Component {
   }
 
   get showEvCert () {
-    if (this.props.titleMode) {
+    if (this.props.titleMode || this.props.isActive) {
```
Comment (diracdeltas) [inline app/renderer/components/navigation/urlBar.js]: awesome, this simplifies things a lot :)

Comment (jasonrsadler) [reply app/renderer/components/navigation/urlBar.js]: Can you check my squash and rebase?

File: app/renderer/components/preferences/payment/ledgerRecovery.js [hunk 6]
```diff
@@ -31,8 +31,10 @@ class LedgerRecoveryContent extends ImmutableComponent {
     this.props.handleRecoveryKeyChange(e.target.value)
   }
 
-  clearRecoveryStatus () {
-    this.props.hideAdvancedOverlays()
+  clearRecoveryStatus (success) {
```
Comment (diracdeltas) [inline app/renderer/components/preferences/payment/ledgerRecovery.js]: i think the changes to this and the following files aren't supposed to be here

Comment (jasonrsadler) [reply app/renderer/components/preferences/payment/ledgerRecovery.js]: I went back to far on the squash

Comment (jasonrsadler) [reply app/renderer/components/preferences/payment/ledgerRecovery.js]: @diracdeltas Is this back to normal with the right changes?

**Comment by jasonrsadler** [comment]
I hosed this. Moving to PR #13335

**Pull request closed without merging**